libs/ngrx-toolkit/src/lib/with-data-service.spec.ts

```diff
@@ -414,6 +576,19 @@ const createFlight = (flight: Partial<Flight> = {}) => ({
   },
   ...flight,
 });
+
+let currentCustomFlightId = 0;
+const createFlightWithCustomId = (
+  flight: Partial<FlightWithCustomId> = {}
+): FlightWithCustomId => ({
+  flightId: `${++currentCustomFlightId}`,
```

Comment:
let's to the conversion how it is also suggested in the Google styleguide.
```suggestion
  flightId: String(++currentCustomFlightId),
```